client/my-sites/domains/domain-management/edit/domain-types/mapped-domain-type.jsx

```diff
@@ -88,20 +99,62 @@ class MappedDomainType extends React.Component {
 		return (
 			<React.Fragment>
 				<div>
-					<p>{ primaryMessage }</p>
-					{ ! isSubdomain( domain.name ) && (
-						<ul className="mapped-domain-type__name-server-list">
-							{ WPCOM_DEFAULT_NAMESERVERS.map( ( nameServer ) => {
-								return <li key={ nameServer }>{ nameServer }</li>;
-							} ) }
-						</ul>
-					) }
+					<p>{ setupInstructionsMessage }</p>
+					{ this.renderRecommendedSetupMessage( primaryMessage ) }
+					{ domain.aRecordsRequiredForMapping && this.renderARecordsMappingMessage() }
 				</div>
 				<div className="mapped-domain-type__small-message">{ secondaryMessage }</div>
 			</React.Fragment>
 		);
 	}
 
+	renderLinkTo( url ) {
+		return <a href={ url } target="_blank" rel="noopener noreferrer" />;
+	}
+
+	renderRecommendedSetupMessage( primaryMessage ) {
+		const { domain, translate } = this.props;
+
+		return (
+			<FoldableFAQ
+				id="recommended-mapping-setup"
+				question={ translate( 'Recommended setup' ) }
+				expanded
+			>
+				<p>{ primaryMessage }</p>
+				{ ! isSubdomain( domain.name ) && (
+					<ul className="mapped-domain-type__name-server-list">
+						{ WPCOM_DEFAULT_NAMESERVERS.map( ( nameServer ) => {
+							return <li key={ nameServer }>{ nameServer }</li>;
+						} ) }
+					</ul>
+				) }
+			</FoldableFAQ>
+		);
+	}
+
+	renderARecordsMappingMessage() {
+		const { domain, translate } = this.props;
+
+		const advancedSetupUsingARecordsTitle = translate( 'Advanced setup using root A records' );
+		const aRecordsSetupMessage = translate(
+			'We recommend using WordPress.com’s name servers to map your domain, but if you prefer you can use different name servers and manage the configuration of your domain yourself. To point your domain to WordPress.com, please add the following IP addresses as root A records using {{link}}these instructions{{/link}}:',
```

Comment:
I like the wording/warning from the linked support page (https://wordpress.com/support/domains/map-existing-domain/#using-a-records):

> If you map a domain using A records rather than WordPress.com name servers, you will need to manage your domain’s DNS records yourself for any other services you are using with your domain, including email forwarding or email hosting (i.e. with Google Workspace or Titan).

What do you think about adding it as a warning notice like this:

<img width="661" alt="Screenshot 2021-03-10 at 14 52 31" src="https://user-images.githubusercontent.com/3392497/110648142-5da12080-81b0-11eb-8cdb-e9b4d8b98841.png">

Then the instructions themselves can be shorter - and we stress why we recommend the nameserver route. I've also changed `add` to `set`, because someone might interpret that as "keep the existing root A records and _add_ these additional ones".

Reply:
I have mixed feelings about using a notice like that. They are usually located above the content.